councilmatic_core/management/commands/import_data.py

```diff
@@ -355,12 +351,12 @@ def grab_organizations(self):
     def grab_organization_posts(self, org_dict):
         url = base_url + '/organizations/'
 
-        r = session.get(url, params=org_dict)
+        r = session.get(url, params=org_dict, timeout=60)
```

Comment:
Same. Or keep reading.

Reply:
Meh - okaaaayy.

I did not wrap all instances of session.get in the custom `_get_response` function - in part, because we've not encountered funny business with these other pages (it seems like the boundaries and the people images have been the offenders in the past). But also in part, because I get nervous about bleeding edge changes in `import_data`. 

I suppose I should throw caution to the wind and wrap at instances of `session.get` in the new function. We'll try it out in Chicago and see how life changes.